feat: add documentation for showing verification after registration (#1349)
ory/docs: docs/kratos/self-service/flows/user-settings.mdx

}
```

## Show verification form after updating a verifiable address

:::info

Showing the verification form after a settings update is currently only supported on native or SPA clients.

:::

> For SPA & native clients, the response from the registration endpoint contains a `continue_with` field. This field lists possible
> actions, the user might need to or can take next. For example, an object containing the ID of the verification flow created as
> part of the settings flow:

Reviewer: ```suggestion
For SPA and native clients, the response from the registration endpoint contains a `continue_with` field. This field lists possible
actions the user might need to take next. For example, an object containing the ID of the verification flow created as
part of the settings flow:
```

Author: Done.
